crates/sncast/tests/data/scripts/script_manual_tests/Scarb.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "test_script"
+version = "0.1.0"
+
+# See more keys and their definitions at https://docs.swmansion.com/scarb/docs/reference/manifest.html
+
+[dependencies]
+starknet = ">=2.3.0"
+sncast_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.13.1" }
+map = { path = "../../contracts/map" }
+constructor_with_params = { path = "../../contracts/constructor_with_params" }
```

Comment:
if you intend to run it on devnet in some automated tests (which I strongly suggest), please do not use those contracts directly as it may break things. We declare those in out test codebase, and one contract cannot be declared multiple times. It is the best practice to add custom contracts for your new test script, that only this script uses.